src/mentors.json

```diff
@@ -2988,5 +2988,34 @@
         "id": "josectobar"
       }
     ]
+  },
+  {
+    "id": "jv.quilichini@gmail.com",
+    "name": "Jean-Vincent QUILICHINI",
+    "avatar": "https://cloudinary-a.akamaihd.net/hopwork/image/upload/h_360,w_360,c_thumb,g_face,z_0.4,q_auto,dpr_2.0/inxdc4oivbgexiyo7u4r.png",
+    "title": "Senior Fullstack Developer",
+    "description": "Hello 👋 ! Always here & happy to help 🤗",
+    "country": "EU",
+    "tags": [
+      "javascript",
+      "nodejs",
+      "reactjs",
+      "vuejs",
+      "fullstack"
```

Comment:
I replaced it to `fullstack` - this is the most common use :)

Reply:
thanks! hard to resolve conflicts as a new one has just been added the same time I forked the master 😅